providers/src/airflow/providers/databricks/hooks/databricks_sql.py

```diff
@@ -180,6 +196,7 @@ def run(
         self,
         sql: str | Iterable[str],
         autocommit: bool = ...,
+        execution_timeout: timedelta | None = ...,
```

Comment:
It would be better for us to add it as the last argument or keyword only argument to avoid breaking change